tools/lint/tests/test_file_lints.py

```diff
@@ -145,6 +145,51 @@ def test_setTimeout():
                                1)]
 
 
+def test_eventSender():
+    error_map = check_with_files(b"<script>eventSender.mouseDown()</script>")
+
+    for (filename, (errors, kind)) in error_map.items():
+        check_errors(errors)
+
+        if kind == "python":
+            assert errors == [("PARSE-FAILED", "Unable to parse file", filename, 1)]
```

Comment:
Kinda silly to run these checks on python too, but that's all my doing, so I can't complain.